src/main/java/io/gravitee/repository/management/api/search/ApiKeyCriteria.java

```diff
@@ -52,13 +57,23 @@ public boolean isIncludeRevoked() {
         return includeRevoked;
     }
 
+    public long getExpireAtAfter() {
+        return expireAtAfter;
+    }
+
+    public long getExpireAtBefore() {
+        return expireAtBefore;
+    }
+
     public static class Builder {
         private Collection<String> plans;
 
         private long from, to;
 
         private boolean includeRevoked;
 
+        private long expireAtAfter, expireAtBefore;
```

Comment:
I would have call them `expireAfter` and `expireBefore` that I think more human readable / fluent

I had to read the queries to understand the meaning of these fields.